<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAmpSpectrumAddZerosClass</class>
 <widget class="QDialog" name="CAmpSpectrumAddZerosClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>848</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>振幅谱补零</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="InputParGroupBox">
     <property name="title">
      <string>输入信号参数</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="Par0Label">
        <property name="text">
         <string>频率成份数目M：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="Par0SpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="Par1Label">
        <property name="text">
         <string>A1：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="Par1DoubleSpinBox">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="Par2Label">
        <property name="text">
         <string>f1(Hz)：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QDoubleSpinBox" name="Par2DoubleSpinBox">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="Par3Label">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>A2：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="Par3DoubleSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="Par4Label">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>f2(Hz)：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QDoubleSpinBox" name="Par4DoubleSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="Par5Label">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>A3：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="Par5DoubleSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="Par6Label">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>f3(Hz)：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QDoubleSpinBox" name="Par6DoubleSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="Par7Label">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>A4：</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="Par7DoubleSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="Par8Label">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>f4(Hz)：</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QDoubleSpinBox" name="Par8DoubleSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="Par9Label">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>A5：</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="Par9DoubleSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="Par10Label">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>f5(Hz)：</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QDoubleSpinBox" name="Par10DoubleSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="Par11Label">
        <property name="text">
         <string>采样点数N：</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QDoubleSpinBox" name="Par11DoubleSpinBox">
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="Par12Label">
        <property name="text">
         <string>采样间隔△(ms)：</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QDoubleSpinBox" name="Par12DoubleSpinBox">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="AddZeroParGroupBox">
     <property name="title">
      <string>输入补零参数</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="Par13Label">
        <property name="text">
         <string>前补零个数：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="Par13DoubleSpinBox">
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="maximum">
         <double>100000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="Par14Label">
        <property name="text">
         <string>后补零个数：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QDoubleSpinBox" name="Par14DoubleSpinBox">
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="maximum">
         <double>100000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="Par15Label">
        <property name="text">
         <string>隔档插零数目：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="Par15DoubleSpinBox">
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="maximum">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>输出文件</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="OutputFile1Label">
        <property name="text">
         <string>输出时域合成信号f1(n△)：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="OutputFile1LineEdit"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="OutputFile1PushButton">
        <property name="text">
         <string>. . .</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="OutputFile2Label">
        <property name="text">
         <string>输出不补零信号振幅谱F1(k)：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="OutputFile2LineEdit"/>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="OutputFile2PushButton">
        <property name="text">
         <string>. . .</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="OutputFile3Label">
        <property name="text">
         <string>输出前补零信号f2(n△)：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="OutputFile3LineEdit"/>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="OutputFile3PushButton">
        <property name="text">
         <string>. . .</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="OutputFile4Label">
        <property name="text">
         <string>输出前补零信号振幅谱F2(k)：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="OutputFile4LineEdit"/>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="OutputFile4PushButton">
        <property name="text">
         <string>. . .</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="OutputFile5Label">
        <property name="text">
         <string>输出后补零信号f3(n△)：</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="OutputFile5LineEdit"/>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="OutputFile5PushButton">
        <property name="text">
         <string>. . .</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="OutputFile6Label">
        <property name="text">
         <string>输出后补零信号振幅谱F3(k)：</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="OutputFile6LineEdit"/>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="OutputFile6PushButton">
        <property name="text">
         <string>. . .</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="OutputFile7Label">
        <property name="text">
         <string>输出隔档插零信号f4(n△)：</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="OutputFile7LineEdit"/>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="OutputFile7PushButton">
        <property name="text">
         <string>. . .</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="OutputFile8Label">
        <property name="text">
         <string>输出隔档插零信号振幅谱F4(k)：</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="OutputFile8LineEdit"/>
      </item>
      <item row="7" column="2">
       <widget class="QPushButton" name="OutputFile8PushButton">
        <property name="text">
         <string>. . .</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="CalPushButton">
     <property name="text">
      <string>计算</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="CancelPushButton">
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="amp_spectrum_add_zeros.qrc"/>
 </resources>
 <connections/>
</ui>
